templates/project/.github/workflows/test.yaml.twig

```diff
@@ -99,6 +99,9 @@ jobs:
               if: matrix.variant != 'normal' && !startsWith(matrix.variant, 'symfony/symfony')
               {% verbatim %}run: composer require ${{ matrix.variant }} --no-update{% endverbatim %}
 
+            - name: Allow unstable dependencies
+              run: composer config minimum-stability dev
```

Comment:
Should this be only executed when Symfony `5.4`?

Reply:
It doesnt matter, since we are using prefer-stable , when there is a stable release it will be used, but for sf 5.4 there is no stable release so it will use the dev stability